### ISSUE TITLE
Whohas clear counter missing

### ISSUE THREAD

**Issue by ghost** [TotalFreedom/TotalFreedomMod#1920]
whohas clear looks is not working without the counter (add a counter for cleared items)

**Comment by Wild1145** [comment]
@marcocorriero Is it that it is not working at all right now? Or that it doesnt look like it works without a counter?

**Comment by ghost** [comment]
@Wild1145 is working, but it doesn't say Cleared %amount% of %items% , and it makes a illusion to admins the whohas clear is not working.

**Comment by ghost** [comment]
Yeah.... There was never a counter on whohas. The command was just ment to tell an admin who has a certain item, and smite them as an option...
